### DISCUSSION TITLE
Valid certificate does not seem to be accepted by the Tesla API

### DISCUSSION THREAD

**Discussion by n1thun** [vercel/community#7276] [Help] [answered]
### Summary

So I currently have my nextjs app deployed on vercel and the domain is provided by godaddy, I am trying to setup an integration with the tesla api and am facing an issue where the tesla self service app says that my domain is invalid. I can see that my website has an ssl and a valid certificate. I tried google.com and that seemed to work, any idea whats going on here and what I might be missing. Thanks.

### Steps to Reproduce

<img width="806" alt="image" src="https://github.com/vercel/community/assets/2892349/728de1c6-9cdf-49d8-8210-badd7481c32a">

**Comment by swarnava** [comment, marked as answer]
Hi,

Unfortunately this is an issue on Tesla end and we cannot do much about it: https://github.com/teslamotors/vehicle-command/issues/159. Looks like they are expecting to have `.com` domain. Please contact to their team to find more information on this one.

[a comment by an automated account is not shown]
